benchmark-service/benchmark/benchmark.py

```diff
@@ -269,13 +277,11 @@ async def github_polling_loop(app):
 
 
 async def on_startup(app):
-    with open(os.environ.get('HAIL_CI_OAUTH_TOKEN', 'oauth-token/oauth-token'), 'r') as f:
-        oauth_token = f.read().strip()
     app['gs_reader'] = ReadGoogleStorage(service_account_key_file='/benchmark-gsa-key/key.json')
     app['github_client'] = gidgethub.aiohttp.GitHubAPI(aiohttp.ClientSession(),
                                                        'hail-is/hail',
                                                        oauth_token=oauth_token)
-    app['batch_client'] = await bc.BatchClient(billing_project='benchmark')
+    app['batch_client'] = await bc.BatchClient(billing_project='test')
```

Comment:
This needs to be benchmark

Reply:
I don't have to wait for #9668 to go in?